index.js

```diff
@@ -1,6 +1,4 @@
-module.exports = function (options) {
-  const Noble = require('./lib/noble');
-  const bindings = require('./lib/resolve-bindings')(options);
+const Noble = require('./lib/noble');
+const bindings = require('./lib/resolve-bindings');
```

Comment:
this should be `const bindings = require('./lib/resolve-bindings')();`

Reply:
Or modify the code of the entire file to:

```
module.exports = require('./with-custom-binding')();
```

Reply:
Done